config.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
   enabled: true,
   workingDirectory: process.cwd(),
 
+  // The root directory containing the package.json for your project
+  // inside your git repository. This should be provided if your deployed
+  // application appears as a subdirectory of your git repository.
+  gitRoot: undefined,
```

Comment:
I've taken another stab at the wording here. I think we'll need to document this more completely in the README once we're satisfied with the behavior here.